plugins/inputs/ipset/ipset_entries_test.go

```diff
@@ -0,0 +1,96 @@
+package ipset
+
+import (
+	"testing"
+	"time"
+
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/testutil"
+	"github.com/stretchr/testify/require"
+)
+
+func TestIpsetEntries(t *testing.T) {
+	acc := new(testutil.Accumulator)
```

Comment:
Please use this form
```suggestion
	var acc testutil.Accumulator
```
and move this down just before the first use to limit the scope of this variable. Same for other tests.